ngraph/frontend/paddlepaddle/src/op/uniform_random.cpp

```diff
@@ -0,0 +1,32 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <ngraph/opsets/opset6.hpp>
+#include "uniform_random.hpp"
+
+namespace ngraph {
+    namespace frontend {
+        namespace pdpd {
+            namespace op {
+
+                NamedOutputs uniform_random (const NodeContext& node) {
+                    return node.default_single_output_mapping({std::make_shared<opset6::Parameter>(element::f32, PartialShape{Dimension::dynamic(), Dimension::dynamic(), Dimension::dynamic(), Dimension::dynamic()})}, {"Out"});
```

Comment:
is 'uniform_random' needed? It is absent in op_table list and there is no tests for this

Reply:
Ok, we will delete this since current it's not needed and reimplement it later